CHANGELOG.rst

```diff
@@ -1,3 +1,8 @@
+UNRELEASED - 2018-03-05
+=======================
+
+- Add option to use a specific AWS configuration set
```

Comment:
Can you please modify the UNRELEASED CHANGELOG entry below instead of creating a new section? Perhaps by editing the second bullet to:

```rst
- Add ``AWS_SES_*`` settings to configure AWS credentials and SES
  configuration sets through ``settings.py``.
```